### PR TITLE
style: format with Ruff

### PR DESCRIPTION
**Pull request by jsstevenson**
Per a previous discussion, this PR implements only the formatting half of `ruff`.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -107,3 +104,8 @@ disable = "R0913"
 
 [tool.pylint.format]
 max-line-length = "120"
+
+
+[tool.ruff]
+src = ["src", "test"]
```
Comment (korikuzma) [inline pyproject.toml]:
```suggestion
src = ["src", "tests"]
```

**Review by korikuzma — CHANGES_REQUESTED** [review]
@jsstevenson do you plan on adding a `.git-blame-ignore-revs` file? More [here](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view)

**Comment by jsstevenson** [comment]
@korikuzma sure, the # of changes is fairly minimal but we can

File: pyproject.toml (hunk 1 above)
Comment (korikuzma) [reply pyproject.toml]: @jsstevenson Should this be resolved?

**Comment by korikuzma** [comment]
> @korikuzma sure, the # of changes is fairly minimal but we can

@jsstevenson Ya, I think it'd be good to do for all the other ruff format PRs too

File: pyproject.toml (hunk 1 above)
Comment (korikuzma) [reply pyproject.toml]: This PR is still showing up in https://github.com/pulls/review-requested for me. See above comment

**Review by korikuzma — COMMENTED** [review]
See thread above (trying to remove from my requested reviews)

**Comment by jsstevenson** [comment]
ope didn't commit

**Review by korikuzma — APPROVED** [review]

**Pull request merged**